### ISSUE TITLE
[Feature request] change aspect ratio

### ISSUE THREAD

**Issue by kenderipa** [anilbeesetti/nextplayer#698]
Hello,

could you perhaps add the option to change aspect ratio? I don't mean fill/stretch, but rather add some common aspects, like 4:3, 5:4, 16:9 and so on.

Thanks!

**Comment by drogga** [comment]
The BSPlayer used to have a custom preset in the settings, where I could add 16:10 if I wanted to _(because there are monitors with that aspect ratio, which are more common nowadays, especially in/on laptops and Android runs on varies of devices, even on x86)_

**Comment by drogga** [comment]
The BSPlayer used to have a custom aspect ratios input field in the settings and presets with also 2.35:1 & 16:10.

**Comment by kenderipa** [comment]
> The BSPlayer used to have a custom aspect ratios input field in the settings and presets with also 2.35.1 & 16:10.

Of course, these two presets would also be nice.

**Comment by kenderipa** [comment]
@anilbeesetti, what do you think?

**Comment by Pentaphon** [comment]
I agree we need these common aspect ratios for playing different kinds of movies and other videos that need to be played in certain aspects that are not covered by "Crop" and "Fit". VLC does this the best and I want Next Player to do something similar to the following screenshot. All of these aspect ratios are needed to make the player more useful. 

![Screenshot_20240505-064601](https://github.com/anilbeesetti/nextplayer/assets/49374058/35a8b9b1-53f3-4f5f-8539-b8b5f993ad8f)

**Comment by kenderipa** [comment]
Any progress, @anilbeesetti?

**Comment by kenderipa** [comment]
Oh well. Closing, since I'm using mpv.
